influxdb/content.md

```diff
@@ -1,194 +1,138 @@
+InfluxDB is the time series data platform designed to handle high write and query workloads. Using InfluxDB, you can collect, store, and process large amounts of timestamped data, including metrics and events for use cases such as DevOps monitoring, application metrics, IoT sensors, and event monitoring.
 
-[InfluxDB Documentation](https://docs.influxdata.com/influxdb/latest/)
+Use the InfluxDB Docker Hub image to write, query, and process time series data in [InfluxDB v2](https://docs.influxdata.com/influxdb/v2/) or [InfluxDB v1](https://docs.influxdata.com/influxdb/v1/).
 
-%%LOGO%%
-
-## Using this Image - InfluxDB 2.x
-
-### Quick start
-
-Using this image is pretty easy, but there are a few things you should know.
+For more information, visit https://influxdata.com.
 
--	You should forward TCP port 8086
--	You should mount a volume in /var/lib/influxdb2
-
-```console
-$ docker run \
-      -p 8086:8086 \
-      -v myInfluxVolume:/var/lib/influxdb2 \
-      %%IMAGE%%:latest
-```
-
-After starting the container you can use the web interface at http://localhost:8086/ to setup and customize your Influx database.
-
-Find more about API Endpoints & Ports [here](https://docs.influxdata.com/influxdb/v2.5/reference/api/).
-
-### Configuration
-
-InfluxDB can be configured using a mix of a config file, environment variables, and CLI options. To mount a configuration file and use it with the server, you can use this command to generate the default configuration file:
-
-```console
-$ docker run --rm %%IMAGE%%:2.0 influxd print-config > config.yml
-```
-
-Modify the default configuration, which will now be available under `$PWD`. Then start the InfluxDB container:
-
-```console
-$ docker run -p 8086:8086 \
-      -v $PWD/config.yml:/etc/influxdb2/config.yml \
-      %%IMAGE%%:2.0
-```
-
-Modify `$PWD` to be the directory where you want to store the configuration file.
+%%LOGO%%
 
-Individual config settings can be overridden by environment variables. The variables must be named using the format `INFLUXD_${SNAKE_CASE_NAME}`. The `SNAKE_CASE_NAME` for an option will be the option's name with all dashes (`-`) replaced by underscores (`_`), in all caps.
+# How to use this image for InfluxDB v2
 
-Examples:
+**Quick start**: See the guide to [Install InfluxDB v2 for Docker](https://docs.influxdata.com/influxdb/v2/install/?t=Docker) and get started using InfluxDB v2.
 
-```console
-# Config setting: bolt-path
-INFLUXD_BOLT_PATH=/root/influxdb.bolt
-# Config setting: no-tasks
-INFLUXD_NO_TASKS=true
-# Config setting: storage-wal-fsync-delay
-INFLUXD_STORAGE_WAL_FSYNC_DELAY=15m
-```
+## Start InfluxDB v2 and set up with the UI, CLI, or API
 
-Finally, all config options can be passed as CLI options:
+To start an InfluxDB v2 container that you can then set up using the included UI, the `influx` CLI, or the HTTP API, enter the following command:
```

Comment:
This sentence feels awkward to me. I wonder if it should just focus on starting the container and then include the setup content after the container is started.

```suggestion
To start an InfluxDB v2 container, enter the following command:
```

Reply:
Yeah, I was trying to distinguish from the auto-setup, which I wanted to lead with, but it didn't flow without being able to link to sections.